data/categories.txt

```diff
@@ -1800,7 +1800,7 @@ fi:Kioski
 fr:Kiosque
 de:Kiosk
 hu:Trafik
-it:Chiosco
+it:Edicola|Giornalaio
```

Comment:
Thanks for the contribution! I have few questions:
1. Is Chiosco a completely invalid translation?
2. Can someone (maybe non-native Italian/tourist) type "Chiosco" in the search and expect to find shop=kiosk?
3. First translation here will be visible to users, second and others are used in search and search hints only. Now with your proposed change both shop=kiosk and shop=newsagent will display the same "Edicola" name for any map user. Is it correct behavior? Or it would be better to display different names (then you need to change the first, "visible" translation).

Reply:
> Thanks for the contribution! I have few questions:
> 
>     1. Is Chiosco a completely invalid translation?
> 
>     2. Can someone (maybe non-native Italian/tourist) type "Chiosco" in the search and expect to find shop=kiosk?
> 
>     3. First translation here will be visible to users, second and others are used in search and search hints only. Now with your proposed change both shop=kiosk and shop=newsagent will display the same "Edicola" name for any map user. Is it correct behavior? Or it would be better to display different names (then you need to change the first, "visible" translation).

1) "Chiosco" is the literal translation of "kiosk", but when talking we don't use the word "chiosco" to identify a kiosk like the one defined by OSM (https://wiki.openstreetmap.org/wiki/Tag:shop%3Dkiosk); instead we use the word "chiosco" only to identify a place that sells mainly food and drinks, and we tag these places as cafe or bar, even if they are in a small structure as a kiosk.
This is a "chiosco" (we tag it as cafe or bar etc..):
![chiosco-bar](https://user-images.githubusercontent.com/19877271/121302618-4205fd00-c8fa-11eb-940d-eed618ada34a.jpg)
This is a "Edicola" or "Giornalaio" (we tag it kiosk):
![1507538005_edicola1](https://user-images.githubusercontent.com/19877271/121302791-81344e00-c8fa-11eb-8a88-bbde0ea43c4b.jpg)
This is a "Edicola" or "Giornalaio" (we tag it newsagent):
![Screenshot at 2021-06-09 08-19-48](https://user-images.githubusercontent.com/19877271/121303796-c7d67800-c8fb-11eb-99c5-ba438e64488a.png)
The difference between kiosk and newsagent is that "Use shop=kiosk for a similar shop in a little freestanding kiosk box, e.g. located in the middle of a square, or possibly for a very small box shop as part of a bigger building. This type of outlet is very common in France, Switzerland and Greece. A newsagent is a proper shop, so slightly bigger building or integrated into a bigger building. " (as per https://wiki.openstreetmap.org/wiki/Tag:shop%3Dnewsagent).

2) If they want a place selling food/drinks they'll have to search for cafe or bar, but even when typing "chiosco" they would very probably get some results, since we often put the word "Chiosco" in the name of a cafe or bar that is freestanding.
Here are some examples: 
-https://www.openstreetmap.org/way/244769833#map=19/45.84452/8.81431
-https://www.openstreetmap.org/way/432380034#map=19/44.80590/10.31906
-https://www.openstreetmap.org/node/4662947650#map=19/45.57174/9.57345

3) Yes, that's correct. :)

Reply:
Thanks, I see your point.

But what about other countries? Your translations will be visible everywhere else for these tags. For example, in Zurich, there are many shop=kiosk that also sell light drinks and snacks. Here the same "Edicola" name may confuse Italian tourists, no?

And I think it makes sense to add Chiosko in any case as an invisible search synonym, e.g. for tourists in Italia who may use the word from the dictionary or from school language courses.

Reply:
> For example, in Zurich, there are many shop=kiosk that also sell light drinks and snacks. Here the same "Edicola" name may confuse Italian tourists, no?

Well yes, that could be a bit consufing, but as you said, a "shop=kiosk could also sell light drinks and snacks", that means that if I'm looking for an "Edicola" then I'll find a place selling newspapers AND food/drinks too.
 
> And I think it makes sense to add Chiosko in any case as an invisible search synonym, e.g. for tourists in Italia who may use the word from the dictionary or from school language courses.

Yes it makes sense, it could be added, but it will remain confusing in this scenario too, because when tourists will search for Chiosco then they will get results for shop=kiosk and also for cafes and bars with the word "Chiosco" in their name (and in this case, cafes and bars won't sell newspapers).

Reply:
If the search found some results, it's better to filter them visually (by type), than if the search returns NO results at all.

So let me summarise: I don't like that after this change two distinct features became completely equal for users from textual and search perspectives, but their icons are different.

What do you think about the following idea? (please think about Italian tourists in other countries too).

1. `4Chiosco|4Edicola|4Giornalaio` for shop=kiosk, so it will be textually different from a newsstand, and will be correctly displayed in most countries (where kiosks are selling not only newspapers and magazines). Plus, anyone who types `edic` or `giorn` will get a hint for the search and will find these kiosk types.
2. `4Edicola|4Giornalaio` for shop=newsagent

Reply:
The first word in the line in this file is visible to users.

Can you please `git commit -a --amend --signoff` the final changes?

Reply:
> Can you please `git commit -a --amend --signoff` the final changes?

Done